resources/js/features/games/components/AchievementSetCredits/AchievementAuthorsDisplay/AchievementAuthorsDisplay.tsx

```diff
@@ -0,0 +1,109 @@
+import { type FC, Fragment } from 'react';
+import { useTranslation } from 'react-i18next';
+import { FaTrophy } from 'react-icons/fa';
+
+import {
+  BaseTooltip,
+  BaseTooltipContent,
+  BaseTooltipTrigger,
+} from '@/common/components/+vendor/BaseTooltip';
+import { UserAvatar } from '@/common/components/UserAvatar';
+import { UserAvatarStack } from '@/common/components/UserAvatarStack';
+import { cn } from '@/common/utils/cn';
+
+import { TooltipCreditRow } from '../TooltipCreditRow';
+import { TooltipCreditsSection } from '../TooltipCreditsSection';
+
+interface AchievementAuthorsDisplayProps {
+  authors: App.Platform.Data.UserCredits[];
+}
+
+export const AchievementAuthorsDisplay: FC<AchievementAuthorsDisplayProps> = ({ authors }) => {
+  const containerClassNames = cn(
+    'flex items-center rounded-md bg-amber-900/20 py-1 pr-2',
+    'light:bg-white light:border light:border-amber-300',
+  );
+
+  // 4-6 authors - show avatar images and name labels for top 2, stack for the rest
```

Comment:
Rather than basing this on author count, could it be tied to percentage? Expand users who have contributed at least 33% of the achievements for the set, and collapse everyone else?

For example, on this game: http://localhost:64000/game2/4986 I have one achievement and the primary author has 35. I don't feel like I should be featured prominently for that set.

Reply:
Great idea. I've made this change in latest.

![Screenshot 2025-06-26 at 5 12 52 PM](https://github.com/user-attachments/assets/ad60088f-f190-4cc6-9808-2b0e6cae3407)